src/components/common/YourCompany/YourCompany.vue

```diff
@@ -458,19 +454,14 @@ export default class YourCompany extends Mixins(
     return this.toDisplayPhone(this.nrApplicant.phoneNumber)
   }
 
-  /** The recognition date or business start date-time string. */
-  get recognitionDateTime (): string {
+  /** The recognition date or business start date string. */
+  protected get recognitionDateTime (): string {
     if (this.isBenIaCorrectionFiling) {
       if (this.getCorrectedFilingDate) {
```

Comment:
I'm thinking that BEN IA corrections should also display the business founding date (not the date of the original IA). Please confirm with BA/PO and change accordingly.

Reply:
Okay I am waiting to hear back from Mihai.

Reply:
FWIW, it will still be called Recognition Date and Time for BEN filings.

Reply:
From stand up on Friday I made these changes